endpoints/users/controller.py

```diff
@@ -35,43 +39,45 @@
 
 
 class ListUsersResource(Resource):
+    @jwt_required
     def get(self):
         try:
-            users = User.query.all()
-            users = [marshal(user, user_list_fields) for user in users]
-            return users
-
-        except Exception as error:
-            response.errorResponse(str(error))
-            return response.__dict__
+            users = userRepository.find()
+            return marshal(users, user_list_fields)
+        except:
+            return {'message': 'Something went wrong', 'timestamp': round(time.time())}, 500
 
 
 class RegisterUserResource(Resource):
     def post(self):
         try:
-            user = register_parser.parse_args()
+            credentials = register_parser.parse_args()
+            user = userRepository.findByUsername(credentials['username'])
+            if(bool(user)):
```

Comment:
You can use `if user:`

An empty object resolves to False, by the way.

Reply:
That's what I'm aiming for, but I guess casting as a bool is unnecessary.